src/strategies/index.ts

```diff
@@ -864,7 +866,9 @@ const strategies = {
   'a51-vault-balance': a51VaultBalance,
   'quickswap-v3': quickswapv3,
   'balance-of-with-bazaar-batch-auction-linear-vesting-power':
-    balanceOfWithBazaarBatchAuctionLinearVestingPower
+    balanceOfWithBazaarBatchAuctionLinearVestingPower,
+  'staking-balance-of-v1': stakingBalanceOfV1,
+  'staking-balance-of-v2': stakingBalanceOfV2,
```

Comment:
Could you create separate strategies for these two please. will be easy to review/ integrate :) Thanks